kubeflow/core/iap.libsonnet

```diff
@@ -499,6 +499,7 @@
       spec: {
         rules: [
           {
+            [if hostname != "null" then "host"]: hostname,
```

Comment:
looks weird. guess there is a typo?

Reply:
No, that's correct. I wanted to use the parameter name `hostname` for `iap.jsonnet` component since it is more obvious.

k8s Ingress object uses the `host` key instead of `hostname` in it's spec.

Reply:
I mean, if hostname == "null", this line will render
: null

Shouldn't it be 
[if hostname != "null" then "host": hostname]

Reply:
The syntax `[...]` means that it is a computed field in jsonnet. if hostname == "null", then the field will be omitted.

Look at Computed and Optional Fields section in http://jsonnet.org/docs/tutorial.html

Reply:
Ah I see, thanks